lib/__tests__/build-tokens.test.js

```diff
@@ -63,6 +63,21 @@ describe('buildTokensCommand', () => {
     }));
   });
 
+  it('should handle the exclude-core flag', async () => {
+    await buildTokensCommand(['--exclude-core']);
+
+    // only light theme, not core
+    expect(StyleDictionary).toHaveBeenCalledTimes(1);
```

Comment:
nit: how much effort might it be to have an assertion to verify it was explicitly the light theme vs relying solely on the count?

Reply:
addressed in https://github.com/openedx/paragon/compare/9bdb1ce8a3c783f77e301bd73f916785d95522d1..503833f68d2b0e9c52b9522e5b08e6e53fa9fa15

Reply:
nice, thanks for updating!